bindings/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+enable_language(Fortran)
+find_package(MPI REQUIRED)
```

Comment:
MPI should have been found before

Reply:
not really - it is found before `enable_language(Fortran)`, so `MPI::MPI_Fortran` is not available. Would have to change main `CMakeLists.txt`. OK?

Reply:
I moved `enable_language(Fortran)` to the beginning of main `CMakeLists.txt`, together with CUDA, and before `find_package(MPI)`